### PR TITLE
outline

### PR DESCRIPTION
**Pull request by GregTheGreek**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,26 @@
-# UBIPs
\ No newline at end of file
+# Proposing Changes to Ultralight Beam ("UB")
+## Introduction
+The UB project's development process is design-driven. Significant changes to the language, libraries, or tools must be first discussed, and sometimes formally documented, before they can be implemented.
```
Comment (decanus) [inline README.md]:
```suggestion
The UB project's development process is design-driven. Significant changes to the protocol, libraries, or tools must be first discussed, and sometimes formally documented, before they can be implemented.
```

File: README.md [hunk 2]
```diff
@@ -1 +1,26 @@
...
+  - decline proposal, or
+  - ask for a design doc.
+ 
+  If the proposal is accepted or declined, the process is done. Otherwise the discussion is expected to identify concerns that should be addressed in a more detailed design.
+
+3. The proposal author writes a design doc to work out details of the proposed design and address the concerns raised in the initial discussion.
+
+4. Once comments and revisions on the design doc wind down, there is a final discussion on the issue, to reach one of two outcomes:
+  - Accept proposal or
+  - decline proposal.
```
Comment (ec2) [inline README.md]: Formatting not consistent from point 2.

File: README.md [hunk 3]
```diff
@@ -1 +1,26 @@
-# UBIPs
\ No newline at end of file
+# Proposing Changes to Ultralight Beam ("UB")
+## Introduction
```
Comment (decanus) [inline README.md:2]:
```suggestion

## Introduction
```

**Review by decanus — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (GregTheGreek) [reply README.md]: fixed

File: README.md [hunk 4]
```diff
@@ -5,22 +5,27 @@ The UB project's development process is design-driven. Significant changes to th
...
-  - After the proposal is accepted or declined (whether after step 2 or step 4), implementation work proceeds in the same way as any other contribution.
+     - Accept proposal, or
+     - decline proposal, or
+     - ask for a design doc.
+
+   If the proposal is accepted or declined, the process is done.
+   Otherwise the discussion is expected to identify concerns that
+   should be addressed in a more detailed design.
+
+3. The proposal author writes a [design doc](#design-documents) to work out details of the proposed
```
Comment (decanus) [inline README.md]: Referencing a non existant section

**Review by decanus — DISMISSED** [review]

File: README.md [hunk 5]
```diff
@@ -1 +1,31 @@
-# UBIPs
\ No newline at end of file
+# Proposing Changes to Ultralight Beam ("UB")
+## Introduction
+The UB project's development process is design-driven. Significant changes to the protocol, libraries, or tools must be first discussed, and sometimes formally documented, before they can be implemented.
+
+This document describes the process for proposing, documenting, and implementing changes to the UB project.
+
+## The Proposal Process
+1. The proposal author [creates a brief issue](https://golang.org/issue/new) describing the proposal.\
```
Comment (decanus) [inline README.md]:
```suggestion
1. The proposal author [creates a brief issue](https://github.com/ultralight-beam/UBIPs/issues/new) describing the proposal.\
```

**Review by decanus — CHANGES_REQUESTED** [review]

File: README.md [hunk 6]
```diff
@@ -1 +1,31 @@
...
+   If the proposal is accepted or declined, the process is done.
+   Otherwise the discussion is expected to identify concerns that
+   should be addressed in a more detailed design.
+
+3. The proposal author writes a design doc (TBD) to work out details of the proposed
+   design and address the concerns raised in the initial discussion.
+
+4. Once comments and revisions on the design doc wind down, there is a final
+   discussion on the issue, to reach one of two outcomes:
+    - Accept proposal or
```
Comment (ec2) [inline README.md]: add a comma. otherwise, LGTM

**Review by ec2 — DISMISSED** [review]

File: README.md (hunk 6 above)
Comment (GregTheGreek) [reply README.md]: suggest it plz

Comment (ec2) [reply README.md]:
```suggestion
    - Accept proposal, or
```

**Review by decanus — DISMISSED** [review]

**Review by decanus — APPROVED** [review]

**Review by wemeetagain — APPROVED** [review]

**Review by decanus — APPROVED** [review]

**Pull request merged**